### PR TITLE
Replace 'imgur' with 'fiware'

### LINKED ISSUE #1: Replace 'imgur' with 'fiware'
Some of the files in this repository still mention Imgur. E.g.

- package.js - metadata should be updated
- accounts-imgur.js - should be renamed
    - https://github.com/apinf/apinf-accounts-idm/blob/1f77c9936ce992c287c78447524dd36932921e31/accounts-imgur.js#L6
    - https://github.com/apinf/apinf-accounts-idm/blob/master/accounts-imgur.js#L39
    - https://github.com/apinf/apinf-accounts-idm/blob/master/accounts-imgur.js#L49
- accounts-imgur_login_button.css

Rename the files and change references for Imgur to FIWARE

### PR DESCRIPTION
**Pull request by vellames2**
Closes #1 

The file names and internal code calls have been changed from "Imgur" to "Fiware"

### CODE REVIEW

**Comment by vellames2** [comment]
@mauriciovieira ready for review

File: accounts-fiware_login_button.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
-#login-buttons-image-imgur {
+#login-buttons-image-fiware {
     background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABAklEQVQ4jZ2RMU7DMBSGPwcnjauEuEYgKKgjI+IETJyhC+foSE/SBU7BJRALE2qZw4hgSEliMxVV1Ena/pOH933P7z0xebp9sM5eskdUGr5K6+yVw17vCseJpHcYEOzTOU4kKgsRQrQLBtGQQTRshIFmQRQoxqMp49E9UaC8MIBsEpR2SV7M/94+uFXgsCy+ngHoJYEXbhUciJCbkzu0zpjlL1Tux1vXuAOBQGuNMUcINjt3ClQaYYzxfrtTECeSfsPM/7Oxg9W2a2pKuwSgpt5OsH4q6yoe3ycAWFd1C3x3zovFdiOcmnNS3e8sXk/lKj7LD6TW2dvZ8cVO8CpF/T3/BaxpQTa7TfPPAAAAAElFTkSuQmCC);
```
Comment (mauriciovieira) [inline accounts-fiware_login_button.css]: @Vellames replace this image. or remove it. it's imgur image.

File: package.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 Package.describe({
-  name: 'tunguska:accounts-imgur',
+  name: 'apinf:accounts-fiware',
   version: '0.0.1',
-  summary: 'OAuth2 for Imgur',
-  git: 'https://github.com/robfallows/tunguska-accounts-imgur',
+  summary: 'OAuth2 for Fiware',
```
Comment (mauriciovieira) [inline package.js]: Should mention _FIWARE IdM_ here

**Review by mauriciovieira — CHANGES_REQUESTED** [review]
Please fix these points

File: package.js (hunk 2 above)
Comment (Vellames) [inline package.js]: @mauriciovieira i need the fiware logo to put in package, where i can find a nice image of fiware?

File: accounts-fiware_login_button.css (hunk 1 above)
Comment (vellames2) [reply accounts-fiware_login_button.css]: @mauriciovieira i need the fiware logo to put in package, where i can find a nice image of fiware?

**Comment by vellames2** [comment]
@mauriciovieira or @midyan please review the changes

**Review by mauriciovieira — APPROVED** [review]
LGTM

**Pull request merged**